### PR TITLE
MEPTS-592: Move Tx Curr to Base Cohort in DSD Indicators

### PR DESCRIPTION
**Pull request by ynurmahomed**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java [hunk 1]
```diff
@@ -497,7 +486,7 @@ public CohortDefinition getD2() {
             "endDate=${endDate},location=${location}"));
 
     cd.setCompositionString(
-        "allPatientsTxCurr AND NOT (activeAndStablePatients OR pregnantOrBreastfeedingOrTBTreatment)");
+        "NOT (activeAndStablePatients OR pregnantOrBreastfeedingOrTBTreatment)");
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:489]: @ynurmahomed Think you forgot to remove this NOT, or better, does this works?

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java [hunk 2]
```diff
@@ -771,16 +756,14 @@ public CohortDefinition getN3() {
...
 
     String mappings = "onOrBefore=${endDate},location=${location}";
-    cd.addSearch("TxCurr", EptsReportUtils.map(txCurr, mappings));
     cd.addSearch("patientsEnrolledOnGaac", mapStraightThrough(patientsEnrolledOnGaac));
     cd.addSearch("startOrContinueGAAC", EptsReportUtils.map(startOrContinueGAAC, mappings));
     cd.addSearch("completedGAAC", EptsReportUtils.map(completedGAAC, mappings));
 
     cd.setCompositionString(
-        "TxCurr AND (patientsEnrolledOnGaac OR startOrContinueGAAC) NOT (completedGAAC)");
+        "(patientsEnrolledOnGaac OR startOrContinueGAAC) NOT (completedGAAC OR PregnantAndBreastfeedingAndOnTBTreatment)");
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:766]: PregnantAndBreastfeedingAndOnTBTreatment is missing from the .addSearch

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java (hunk 1 above)
Comment (ynurmahomed) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:489]: It works.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java (hunk 2 above)
Comment (ynurmahomed) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java]: It should be removed (#180).

**Pull request merged**